### PR TITLE
File View added

### PR DESCRIPTION
**Pull request by VamsiP23**
Here's a recording of the viewing pdf functionality: https://streamable.com/g0q0x7
The UI on the file pdf view isn't the greatest, so I can make it better if needed.

### CODE REVIEW

File: CanvasPlusPlayground/Helpers/PDFViewHelper.swift [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+//  Created by Vamsi Putti on 9/17/24.
+//
+
+import SwiftUI
+import PDFKit
+
+struct PDFViewHelper: UIViewRepresentable {
+    let pdfURL: URL
+    
+    init(pdfURL : URL) {
```
Comment (rahulon12) [inline CanvasPlusPlayground/Helpers/PDFViewHelper.swift:14]: nit: This init isn't needed

**Review by rahulon12 — COMMENTED** [review]
A couple of nits, but looks good. Could you attach a quick recording too?

**Comment by VamsiP23** [comment]
Here's a recording: [https://streamable.com/g0q0x7]
The UI on the file pdf view isn't the greatest, so I can make it better if needed.

**Review by rahulon12 — APPROVED** [review]
Nice! LGTM!

**Pull request merged**